README.md

```diff
@@ -41,15 +41,13 @@ The following steps are optional. They're only required if you want to run the w
 
 4. [Install Intel Gaudi device plugin for Kubernetes](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Kubernetes_Installation/index.html).
 
-   Alternatively, Intel provides a base operator to manage the Gaudi software stack. Refer to [this file](kubernetes-addons/Intel-Gaudi-Base-Operator/README.md) for details.
-
```

Comment:
While removing the separate dir & README that just provide link to HabaLabs operator doc is fine, I think the (correct) link would still be useful in this doc:
`Alternatively, Intel provides a [base operator](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Kubernetes_Installation/Kubernetes_Operator.html) to manage the Gaudi software stack.`

Reply:
done.